testsuite/HOWTO.md

```diff
@@ -0,0 +1,176 @@
+`pip install pytest-cov`
+
+Then you can run -
+```
+cd testsuite
+pytest --numprocesses 4 --cov=MDAnalysis --cov-report html
+```
+
+# Details #
+We are borrowing some of NumPy's testing frame work; thus, numpy **must** be installed for the tests to run at all.
+
+## Running tests from within python ##
+Run all the tests with
+```
+python -c 'from MDAnalysisTests import run; run()'
+```
+
+## Running tests from the command line ##
+Instead of running tests from within python, one can also run them via the command line.
+Go into the tests directory (or the package root)
+```
+cd /testsuite/MDAnalysisTests
+```
+and invoke pytest directly to run **all tests** on four processors in parallel ("`%`" is the shell prompt and should _not_ be typed):
+```
+% pytest --numprocesses 4
+```
+Again, to run the tests in parallel (use `--numprocesses 4`) you need to install `pytest-xdist` via `pip install pytest-xdist`
+
+When you have written a **new unit test** it is helpful to check that it passes without running the entire suite. For example, in order to test everything in, say, `test\_selections.py` run
+```
+% pytest test_selections
+..............
+----------------------------------------------------------------------
+Ran 14 tests in 3.421s
+
+OK
+```
+One can also test individual test classes. For instance, after working on the XYZReader one can check just the TestCompressedXYZReader tests with
+```
+% pytest test_coordinates::TestCompressedXYZReader
+....
+----------------------------------------------------------------------
+Ran 4 tests in 0.486s
+
+OK
+```
+where we are testing the class `TestCompressedXYZReader` which can be found in the module (file) `test\_coordinates.py`.
+
+
+### Running tests with setuptools ###
+
+# TODO: Fix further
+
+## Data ##
+The simulation data used in tests are all released under the same license as MDAnalysis or are in the Public Domain (such as PDBs from the Protein Databank). An incomplete list of sources:
+* from Beckstein et al. (2009) (`adk.psf`,`adk_dims.dcd`)
+  * _adk\_dims_      Trajectory of a macromolecular transition of the enzyme adenylate kinase between a closed and an open conformation. The simulation was run in [CHARMM](http://www.charmm.org) c35a1.
+* unpublished simulations (O. Beckstein)
+  * _adk\_oplsaa_    Ten frames from the first 1 ns of a equilibrium trajectory of AdK in water with Na+ counter ions. The OPLS/AA forcefield is used with the TIP4P water model. The simulation was run with [Gromacs](http://www.gromacs.org) 4.0.2.
+* contributions from developers and users
+* Protein Databank
+
+
+### References ###
+
+  * O. Beckstein, E.J. Denning, J.R. Perilla and T.B. Woolf, Zipping and Unzipping of Adenylate Kinase: Atomistic Insights into the Ensemble of Open-Closed Transitions. J Mol Biol 394 (2009), 160--176, doi:[10.1016/j.jmb.2009.09.009](http://dx.doi.org/10.1016/j.jmb.2009.09.009)
+
+
+# Writing test cases #
```

Comment:
The idea is not so much of keeping this file than to copy its content to the wiki. It is a file only because there is no pull request mechanism for the wiki.